libs/header/sync/sync_test.go

```diff
@@ -214,40 +209,6 @@ func TestSyncPendingRangesWithMisses(t *testing.T) {
 	assert.Empty(t, syncer.pending.Head()) // assert all cache from pending is used
 }
 
-// Test that only one objective header is requested at a time
-func TestSyncer_OnlyOneRecentRequest(t *testing.T) {
```

Comment:
This test test is not need anymore, as there is exactly the same and simpler one in sync_getter_test.go